### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by sswathisri** [lula/ngx-summernote#61]
Hi,

It is really cool text editor. Looks like Accessibility is missing. When i run accessible tool, i see below issues.
![image](https://user-images.githubusercontent.com/2986450/68348897-30ba9600-00b0-11ea-97ee-6ce165e1871f.png)
![image](https://user-images.githubusercontent.com/2986450/68348905-39ab6780-00b0-11ea-9e23-deb864517eb7.png)

It would be great if the above issues are fixed. Thanks in advance.

**Comment by lula** [comment]
Hello @sswathisri thanks. This sound more related to summernote rather than this lib, is it? Is summernote standalone accessibility ok?
